ci-templates/.github/workflows/unit-test.yml

```diff
@@ -1,10 +1,9 @@
 name: Testing Suite
 
 on:
-  push:
+  pull_request:
```

Comment:
I like this! The only thing we should acknowledge giving up is the final "I merged my work in, and now production *doesn't* pass tests" 

an incredibly unlikely scenario, but one that could still happen... and arguably one that is the **most** important to find out about.

Reply:
What was happening here is that the build and deploy was running at the same time as a final test run, so it didn't prevent a deploy, but it did cause additional churn on the Actions runners. This change aligns our GitHub Actions approach with our Buddy approach in that we require pull request tests to pass to merge, but after merge, it just deploys.

There are a few things we can do to mitigate the potential situation where tests pass but a recent merge to production would cause them to fail. My recommendation there would be to enforce the rule in GitHub where feature branches must be up to date with the head branch before merge is allowed.